### PR TITLE
Fix SceneTime and add missing pubdate

### PR DESCRIPTION
**Pull request by duramato**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

Fixes SceneTime parsing and adds missing pubdate.
Also adds missing cats.

### CODE REVIEW

File: medusa/providers/torrent/html/scenetime.py [hunk 1]
```diff
@@ -131,9 +132,9 @@ def parse(self, data, mode):
                     continue
 
                 try:
-                    link = cells[labels.index('Name')].find('a')
-                    torrent_id = link['href'].replace('details.php?id=', '').split('&')[0]
-                    title = link.get_text(strip=True)
+                    link = cells[labels.index('Name')]
+                    torrent_id = link.find('a')['href'].replace('details.php?id=', '').split('&')[0]
+                    title = link.find('a').get_text(strip=True)
```
Comment (medariox) [inline medusa/providers/torrent/html/scenetime.py]: Why this change?

File: medusa/providers/torrent/html/scenetime.py [hunk 2]
```diff
@@ -112,7 +113,7 @@ def parse(self, data, mode):
         items = []
 
         with BS4Parser(data, 'html5lib') as html:
-            torrent_rows = html.find_all('tr')
+            torrent_rows = html.find('div', {'id": 'torrenttable'}).find_all('tr')
```
Comment (medariox) [inline medusa/providers/torrent/html/scenetime.py]: I think the preferred syntax is: `.find('div', id='torrenttable')`

File: medusa/providers/torrent/html/scenetime.py (hunk 1 above)
Comment (duramato) [reply medusa/providers/torrent/html/scenetime.py]: That way I can use the `link`, already defined before, for the lineage. And isn't defined again twice.

File: medusa/providers/torrent/html/scenetime.py (hunk 2 above)
Comment (sharkykh) [reply medusa/providers/torrent/html/scenetime.py]: Also note the quotation marks don't match.

Comment (duramato) [reply medusa/providers/torrent/html/scenetime.py]: Changed.

File: medusa/providers/torrent/html/scenetime.py (hunk 1 above)
Comment (medariox) [reply medusa/providers/torrent/html/scenetime.py]: I see it now, all good.
